src/Pos/Ssc/GodTossing/Toss/Class.hs

```diff
@@ -57,6 +61,10 @@ class (Monad m, WithLogger m) =>
         m SharesMap
     getShares = lift getShares
 
+    default getVssCertData :: (MonadTrans t, MonadTossRead m', t m' ~ m) =>
+        m VssCertData
+    getVssCertData = lift getVssCertData
+
     default getVssCertificates :: (MonadTrans t, MonadTossRead m', t m' ~ m) =>
         m VssCertificatesMap
     getVssCertificates = lift getVssCertificates
```

Comment:
@rockbmb you could make default definition: `getVssCertificates = certs <$> getVssCertData`